src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,3 +1,4 @@
+#if !NETSTANDARD2_0
```

Comment:
@fredericDelaporte I got it mostly working.  Do you have any idea how to implement `ITransactionPromoter.Promote()` or if it even needs implemented?  It's part of the `IPromotableSinglePhaseNotification` interface.